build/build-runtimeimages-bases.sh

```diff
@@ -49,18 +49,6 @@ fi
 
 labels="--label com.microsoft.oryx.git-commit=$GIT_COMMIT --label com.microsoft.oryx.build-number=$BUILD_NUMBER"
 
-generateDockerFiles=$(find $runtimeImagesSourceDir -type f -name "generateDockerfiles.sh")
```

Comment:
I am not clear why this change is made

Reply:
It's actually pretty clear... I screwed up :)